src/Command/AddUserCommand.php

```diff
@@ -68,8 +72,7 @@ public function __construct(EntityManagerInterface $em, UserPasswordEncoderInter
     protected function configure()
     {
         $this
-            // a good practice is to use the 'app:' prefix to group all your custom application commands
-            ->setName('app:add-user')
+            // you don't need to call ->setName() for configuring the command when it is lazy.
```

Comment:
this comment is useless (people won't see setName at all)

Reply:
removed.